cargo-shuttle/tests/integration/init.rs

```diff
@@ -48,11 +47,10 @@ async fn non_interactive_rocket_init() {
 
     let args = ShuttleArgs::parse_from([
         "cargo-shuttle",
-        "--api-url",
-        "http://shuttle.invalid:80",
         "init",
         "--api-key",
         "dh9z58jttoes3qvt",
+        "--force-name",
```

Comment:
Rather create a new test instead of updating the current ones since I think the old tests should still exist too.

Reply:
The tests break without this, since it will try to verify the name without this flag. This flag also removes the need to use a dummy API url